tools/devops/automation/templates/release/vs-insertion-prep.yml

```diff
@@ -103,13 +107,15 @@ stages:
           patterns: |
             *.nupkg
 
-      - task: NuGetCommand@2
-        displayName: Publish Nugets to dotnet8
+      - task: 1ES.PublishNuget@1
+        displayName: Publish Nugets package
         inputs:
-          command: push
+          useDotNetTask: false # The default is false to use the NuGetCommand task. Set to true to use the DotNetCoreCLI task to publish packages.
           packagesToPush: '$(Build.SourcesDirectory)/package/*.nupkg'
+          packageParentPath: '$(Build.SourcesDirectory)/package'
           nuGetFeedType: external
-          publishFeedCredentials: dnceng-dotnet8
```

Comment:
Which feed do we publish to now? It doesn't look like that's chosen anywhere.

Reply:
yeah, I requested a force insert to check that